perfrunbook/system-load-and-compute-headroom.md

```diff
@@ -1,120 +1,68 @@
 # System Load and Compute Headroom
 
-From an economical perspective compute resources should be utilized fully.
-Ideally the system's performance should increase linearly up to 100% CPU load.
-This can be in the context of pure compute or peak load and fail-over resilience.
-Graviton instances and x86-type instances, such as c6i, behave fundamentally different under load.
-Any initial x86 advantage over Graviton starts to disappear beyond 50% system load.
-This effect is caused by symmetric multithreading and the Linux kernel's preferrence of CPU cores over CPU threads.
-In the following this behavior is investigated with a few illustrative workloads.
+Ideally, a system's performance should scale linearly as more vCPUs are used, up to 100% CPU load.
+This ideal is not always realized due to how vCPUs are implemented on x86 and Graviton processors.
```

Comment:
This ideal is not always realized due to how vCPUs are implemented on our x86 and Graviton based EC2 instances.

Reply:
done